### DISCUSSION TITLE
.returning() is not supported by mysql and will not have any effect. Last version packages

### DISCUSSION THREAD

**Discussion by ewertonrfd** [adonisjs/core#3628] [Help] [answered]
Hi, everyone
I have the following messages on the screen
It doesn't seem to get in the way, but it's annoying lol

**Problema**
<img width="856" alt="2022-03-14" src="https://user-images.githubusercontent.com/13201575/158280713-93168a47-0a7d-410b-832b-1d1ea33a6fe3.png">

Packages versions
<img width="316" alt="2022-03-14 (2)" src="https://user-images.githubusercontent.com/13201575/158280825-49a29df7-008f-4753-8c22-55943730b12e.png">

Mysql
<img width="416" alt="2022-03-14 (1)" src="https://user-images.githubusercontent.com/13201575/158280947-c9b53f9b-4886-4c89-8645-8c4ea096dc3b.png">

**Any tips or is it a bug/warning ?**
Thanks in advance

**Comment by RomainLanz** [comment, marked as answer]
Hey @EwertonDutra! 👋🏻 

We already [have a PR on same](https://github.com/adonisjs/lucid/pull/813).
It should be fixed once it is merged.

**Reply by donald23onno** [reply]
It seems that PR was closed unfortunately due to not passing tests on PostgreSQL?

**Reply by thetutlage** [reply]
It has been fixed in this commit https://github.com/adonisjs/lucid/commit/4ce5fbb8a2f8bbbac53d5e03881fde47283ffaa3. Please use the latest version of Lucid

**Reply by ewertonrfd** [reply]
Hello @thetutlage 
I would like to give feedback...

I just tested with the latest version and it still has the same message
1st I updated in a project that I had already created
2° now I tested in a new project, I just configured lucid and created the base, then when running the seed it gave the message

![image](https://user-images.githubusercontent.com/13201575/161780335-4b3b7641-b89d-4476-85c4-5a2c67c06626.png)

<img width="863" alt="image" src="https://user-images.githubusercontent.com/13201575/161780738-538b59f4-e056-4d46-b695-4cd615f79b9d.png">


From what I followed on the thread, it seems that the commit was not accepted because it didn't pass a test, but it "seems" like it was just some detail

**Reply by thetutlage** [reply]
Hey! Thanks for the feedback. Can you please help reproduce this warning by creating a repo and share it with me? Because, I cannot reproduce it

**Reply by ewertonrfd** [reply]
Of course!

https://github.com/EwertonDutra/showing-warning-lucid
And have a video showing too 👍

**Reply by thetutlage** [reply]
Thanks for the repo. Fixed in `17.2.0` and personally confirmed it works fine this time

**Reply by ewertonrfd** [reply]
Yes, it's working 👏

<img width="711" alt="2022-04-08" src="https://user-images.githubusercontent.com/13201575/162427615-561ffa6a-2865-48ed-a2fb-1b7a2816d687.png">

Working without warnings 😎

**Reply by thetutlage** [reply]
NICE 🤩
